docs/serving/samples/hello-world/helloworld-java-spring/README.md

```diff
@@ -35,15 +32,15 @@ cd knative-docs/docs/serving/samples/hello-world/helloworld-java-spring
    unzip helloworld.zip
    ```
 
-   If you don't have curl installed, you can accomplish the same by visiting the
+   If you don't have `curl` installed, you can accomplish the same by visiting the
    [Spring Initializr](https://start.spring.io/) page. Specify Artifact as
    `helloworld` and add the `Web` dependency. Then click `Generate Project`,
    download and unzip the sample archive.
```

Comment:
For things that have an interface, can we use **Bold** for things like buttons instead of `code` formatting to differentiate? e.g. **Generate Project**

Maybe add it to the style guide?

Reply:
added to ice box and will change here